kale/predict/class_domain_nets.py

```diff
@@ -59,6 +60,29 @@ def n_classes(self):
         return self._n_classes
 
 
+# Previously PredictionHead
+class ClassNet(nn.Module):
+    """
+    Simple classification prediction-head block to plug ontop of the 4D
+    output of a CNN.
+    Args:
+        n_class (int, optional): the number of different classes that can be predicted. Defaults to 10.
+        input_shape (tuples, optional): the shape that input to this head will have. Expected
+                      to be (batch_size, channels, height, width). Defaults to (-1, 64, 8, 8).
+    """
+
+    def __init__(self, n_class=10, input_shape=(-1, 64, 8, 8)):
+        super(ClassNet, self).__init__()
+        self.avgpool = nn.AvgPool2d(input_shape[2])
+        self.linear = nn.Linear(input_shape[1], n_class)
+
+    def forward(self, x):
+        x = self.avgpool(x)
+        x = torch.flatten(x, 1)
+        x = self.linear(x)
+        return x
+
+
 # Previously DataClassifierDigits
```

Comment:
No need. Consider removing it.

Reply:
Done, I have removed all "previously xxx" comments in this file.